src/pages/settings/settings.ts

```diff
@@ -182,12 +198,53 @@ export class SettingsPage {
   }
 
   public openHelpExternalLink(): void {
-    let url = this.appName == 'Copay' ? 'https://github.com/bitpay/copay/issues' : 'https://help.bitpay.com/bitpay-app';
+    let url =
+      this.appName == 'Copay'
+        ? 'https://github.com/bitpay/copay/issues'
+        : 'https://help.bitpay.com/bitpay-app';
     let optIn = true;
     let title = null;
-    let message = this.translate.instant('Help and support information is available at the website.');
+    let message = this.translate.instant(
+      'Help and support information is available at the website.'
+    );
     let okText = this.translate.instant('Open');
     let cancelText = this.translate.instant('Go Back');
-    this.externalLinkProvider.open(url, optIn, title, message, okText, cancelText);
+    this.externalLinkProvider.open(
+      url,
+      optIn,
+      title,
+      message,
+      okText,
+      cancelText
+    );
+  }
+
+  private openLockModal(): void {
+    let config: any = this.configProvider.get();
+    let lockMethod =
+      config && config.lock && config.lock.method
+        ? config.lock.method.toLowerCase()
+        : null;
+    if (!lockMethod || lockMethod == 'disabled')
+      this.navCtrl.push(LockPage, {}, { animate: false });
```

Comment:
The fact that there's no animation here feels a bit weird on iOS. Simplifying to `this.navCtrl.push(LockPage);` will make the page transition feel more native on iOS.